tests/integration/telemetry/stackdriver/stackdriver_filter_test.go

```diff
@@ -71,13 +72,14 @@ func getEchoNamespaceInstance() namespace.Instance {
 	return echoNsInst
 }
 
-func unmarshalFromTemplateFile(file string, out proto.Message) error {
+func unmarshalFromTemplateFile(file string, out proto.Message, index int) error {
 	templateFile, err := ioutil.ReadFile(file)
 	if err != nil {
 		return err
 	}
 	resource, err := tmpl.Evaluate(string(templateFile), map[string]interface{}{
 		"EchoNamespace": getEchoNamespaceInstance().Name(),
+		"ClusterIndex":  fmt.Sprintf("%d", index),
```

Comment:
Would clusterName be more human readable? If we ever set the name from outside the test framework (in some custom install scenario) it might be more readable to use that when debugging/comparing values in pilot logs.

Reply:
+1 ... @stevenctl we should add that capability to 1.9 as well. Do we have a tracking issue for that?

Reply:
Fixed